tests/ResourceSyncingUsingPolymorphicTest.php

```diff
@@ -72,6 +72,8 @@
 
     $centralUser->tenants()->attach('t1');
 
+    expect($centralUser->tenants()->count())->toBe(1); // Assert `tenants` are accessible
```

Comment:
In all of these assertions, probably assert that it's also the right tenant, not just the count.